src/app/lib/layoutGrid.js

```diff
@@ -13,36 +13,43 @@ import {
 export const layoutGridWrapper = css`
   display: grid;
   @media (max-width: ${group3ScreenWidthMin}) {
-    grid-gap: ${GEL_SPACING}; /* Between grid items */
-    padding: 0 ${GEL_SPACING}; /* On grid wrapper */
+    grid-gap: 0 ${GEL_SPACING}; /* Between grid items */
   }
   @media (min-width: ${group3ScreenWidthMin}) {
-    grid-gap: ${GEL_SPACING_DBL}; /* Between grid items */
-    padding: 0 ${GEL_SPACING_DBL}; /* On grid wrapper */
+    grid-gap: 0 ${GEL_SPACING_DBL}; /* Between grid items */
   }
   @media (max-width: ${group3ScreenWidthMax}) {
-    grid-template-columns: repeat(6, 1fr);
+    grid-template-columns: repeat(6, [col] 1fr);
   }
   @media (min-width: ${group4ScreenWidthMin}) and (max-width: ${group4ScreenWidthMax}) {
-    grid-template-columns: repeat(8, 1fr);
+    /* 
+      Math: (1008 - (7*16))/8 = 112
+      Explainer: (width - 7 gaps of 16) / 8 columns = single column width
+    */
+    grid-template-columns: [start] 1fr repeat(8, [col] minmax(0, 7rem)) 1fr [end];
   }
   @media (min-width: ${group5ScreenWidthMin}) {
-    grid-template-columns: repeat(10, 1fr);
+    /* 
+      Math: (1280 - (9*16))/10 = 113.6
+      Explainer: (width - 9 gaps of 16) / 10 columns = single column width
+    */
+    grid-template-columns: [start] 1fr repeat(10, [col] minmax(0, 7.1rem)) 1fr [end];
   }
 `;
 
 export const layoutGridItem = css`
   @media (max-width: ${group2ScreenWidthMax}) {
     grid-column: 1 / -1;
+    padding: 0 ${GEL_SPACING};
   }
   @media (min-width: ${group3ScreenWidthMin}) and (max-width: ${group3ScreenWidthMax}) {
-    grid-column: 2 / -2;
+    grid-column: col 2 / span 4;
```

Comment:
Is there a reason why you changed it to include spans? The reason for having the `2 /-2` syntax was to show the symmetry.

Reply:
No particular reason, happy for this to be returned to `2 / -2`. However as talked about offline this will change to `3 / -3` due to the 2 additional being added either side of the 1280 grid. Or if we are to find a more appropriate name for `col` (EG: `article-col`) this could become `article-col 2 / article-col -2`

Reply:
I agree that `article-col 2 / article-col -2` would be more readable. However, it will make the CSS more verbose. I believe it would be better to keep it as `3 / -3`, so long as we add documentation explaining our breakpoints and column layouts, including a labelled diagram.